Sources/Wendy/cli/commands/OSCommand.swift

```diff
@@ -541,6 +542,161 @@ struct OSCommand: AsyncParsableCommand {
     }
 }
 
+// MARK: - Cache utilities
```

Comment:
shouldn't all of these need to live in the CacheCommand? they are private and seem to be used only there